### PR TITLE
Add SBOM generation in release.yml

### PR DESCRIPTION
**Pull request by xlin7799**

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 name : "coreHTTP"
 version: "v2.1.0"
-description: |
-  "Client implementation of the HTTP/1.1 specification for embedded devices.\n"
+description: "Client implementation of the HTTP/1.1 specification for embedded devices."
+license: "MIT"
+
```
Comment (archigup) [inline manifest.yml:5]: There is an extra line here.

File: manifest.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 dependencies:
   - name : "llhttp"
     version: "release/v6.0.5"
+    license: "MIT"
     repository:
       type: "git"
-      url: "https://github.com/nodejs/llhttp"
-license: "MIT"
+      url: "https://github.com/nodejs/llhttp.git"
```
Comment (archigup) [inline manifest.yml:12]: Why was .git added here?

Comment (xlin7799) [reply manifest.yml:12]: ".git" was added by Paul for the corePKCS11 manifest.yml file, so for consistency I added ".git" for the rest of the libraries as well.

File: manifest.yml (hunk 1 above)
Comment (xlin7799) [reply manifest.yml]: Fixed

**Review by archigup — APPROVED** [review]

**Review by paulbartell — APPROVED** [review]

**Pull request merged**